projects/wave-core-new/src/lib/operators/dialogs/mean-raster-pixel-values-over-time-dialog/mean-raster-pixel-values-over-time-dialog.component.html

```diff
@@ -0,0 +1,53 @@
+<wave-sidenav-header>Temporal Raster Mean Plot</wave-sidenav-header>
+
+<wave-dialog-help>
+    <p>
+        This operator generates a <i>Temporal Raster Mean Plot</i> based on a raster data set (<i>Input</i>).
+        Select the <i>Time Tick Position</i>, which specifies the position of the x-axis ticks,
+        e.g. at the the time start, end or center of the raster's validity.
+        The plot shows an area chart of the mean pixel values (y-axis) of the rasters.
+        The x-axis displays the timestamps.
+        <i>Fill area under curve</i> allows modifying whether to fill the area under the curve.
+        Edit the <i>Output Name</i> of the plot, if desired, and click on <i>Create</i> to generate the plot.
+    </p>
+    <p>
+        Note: Plots change dynamically with the map view and include only pixels or features, which are located in this area.
+        For the display it is recommended to select a time interval and not a single point in time.
+    </p>
+</wave-dialog-help>
+
+<form [formGroup]="form" (ngSubmit)="add()" fxLayout="column">
+    <div fxFlex>
+
+        <wave-layer-selection [types]="inputTypes" formControlName="layer"></wave-layer-selection>
+        <wave-dialog-section-heading title="Config" subtitle="Specify the Histogram"></wave-dialog-section-heading>
+        <mat-form-field>
+            <mat-select placeholder="Time Tick Position" formControlName="timePosition">
+                <mat-option *ngFor="let timePosition of timePositionOptions"
+                            [value]="timePosition.value">{{timePosition.label}}</mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field>
+            <mat-select placeholder="Fill area under curve" formControlName="area">
```

Comment:
wouldn't a checkbox be more apropriate for a binary choice?

Reply:
2cbf586